internal/extender/overhead.go

```diff
@@ -120,32 +135,24 @@ func (o *OverheadComputer) compute(ctx context.Context) {
 			// pending pod or pod succeeded or failed
 			continue
 		}
-		node, err := o.nodeLister.Get(p.Spec.NodeName)
-		if err != nil {
-			svc1log.FromContext(ctx).Warn("node does not exist in cache", svc1log.SafeParam("nodeName", p.Spec.NodeName))
+
+		// found pod with no associated resource reservation, add to overhead
+		if err := o.addPodResourcesToGroupResources(ctx, rawOverhead, p); err != nil {
+			svc1log.FromContext(ctx).Warn("could not add pod resources to overhead, skipping", svc1log.SafeParam("failedPod", p.Name), svc1log.Stacktrace(err))
 			continue
```

Comment:
I think it would be a clearer factoring if you did the error-prone thing (getting the node for the pod) separately, so like:

```golang
node, err = getNodeForPod(p);
if err {
    log.Warn("could not get node for pod, skipping")
}

// code to add pod to overhead should not have error handling
```

Reply:
Addressed 👍